website/docs/r/sentinel_alert_rule_fusion.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_alert_rule_fusion"
+description: |-
+  Manages a Sentinel Fusion Alert Rule.
+---
+
+# azurerm_sentinel_alert_rule_fusion
+
+Manages a Sentinel Fusion Alert Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-workspace"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "pergb2018"
+}
+
+resource "azurerm_sentinel_alert_rule_fusion" "example" {
+  name                       = "example-fusion-alert-rule"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+  alert_rule_template_name   = "f71aba3d-28fb-450b-b192-4e76a83015c8"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
+
+* `log_analytics_workspace_id` - (Required) The ID of the Log Analytics Workspace this Sentinel Fusion Alert Rule belongs to. Changing this forces a new Sentinel Fusion Alert Rule to be created.
+
+* `alert_rule_template_name` - (Required) The name of the alert rule template which is used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
```

Comment:
if this can only ever be a guid shouldn't it be
```suggestion
* `alert_rule_template_id` - (Required) The name of the alert rule template which is used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
```
or
```suggestion
* `alert_rule_template_guid` - (Required) The name of the alert rule template which is used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
```

Reply:
alert_rule_template_id likes a resource id. So I suggest to use alert_rule_template_guid to avoid confusion.